### ISSUE TITLE
Week 8 Goals

### ISSUE THREAD

**Issue by mlkhorn** [mlkhorn/ftc_cosore#16]
### Abstract (Monday)
- [x] Remember that abstract is due Tuesday!
- [x] Take ideas reading Julia's abstract and personal brainstorming and turn them into an outline for the abstract
- [x]  Try to have at least the first half of the abstract written by your meeting with Ben
- [x]  Meet with Ben @12:30 and get his feedback on your outline and the first half of your abstract
- [x]  Incorporate Ben's feedback into what you have so far 
- [x]  Finish writing the second half your abstract by SULI check-in meeting
- [x] Ask Steph and Skylar for their feedback on your abstract in the check-in meeting AND/OR ask Steph to review your abstract in google docs and add comments
- [x] Begin incorporating these comments into your work, possibly finishing by the end of the work day

### Abstract (Tuesday)
- [x] Finish incorporating whatever comments from the SULI check-in meeting you weren't able to address on Monday
- [x]  Send abstract to Steph AND/OR Ben for final review
- [x]  Incorporate any last minute comments
- [x] Send abstract to information release

### Code I Wrote on Friday To Calculate Annual Rs, Winter Rs, and Winter Rs Contribution
- [x] Add some comment love! On Friday you wrote a bunch of code but didn't have time to write in enough comments. Write comments that help other people understand what you are doing!

### Fix FT-Event Counting Code
- [x]  Right now I have code that counts the number of day scale ft-changes in each dataset, not in each year in each dataset. Fix that. 
- [x]  Add a column to `winterdata_summary` that lists the number of day scale ft-events that occur in each year of each dataset
- [ ] Possibly: Add a column to `winterdata_summary` that lists the number 30 minute scale ft-events that occur in each year of each dataset. These numbers may yield some interesting insights. 

### FT-Events and Winter Rs Contribution Visualization
- [x]  Create a plot that graphs the number of ft-events (x) at the daily scale against the contribution of Winter Rs to annual Rs (y).
- [x]  Improve formatting 1

### FT-Events and Winter Rs Visualization
- [x]  Create a plot that graphs the number of ft-events (x) at the daily scale against Winter Rs (y).
- [x]  Improve formatting 1

### Temperature Visualization (continuing work from last week)
- [x]  Finish creating the column in `plt6_data` that stores a value of TRUE if a daily T5 minimum occurs directly before or after a ft-change at the daily scale 
- [x] Use this column to plot a dataset of ft-change adjacent daily T5 minimums (points colored red) over all the other daily T5 minimums (points colored black), highlighting ft-changes in `plt6`
- [x]  Improve the formatting of this plot 1
- [ ]  Possibly: Create a similar column in `winterdata_bound` that stores a value of TRUE if a temperature measurement was taken directly before/after a ft-change at the thirty minute scale
- [ ]  Possibly: In `plt7` use the new `winterdata_bound` column to highlight every 30 minute scale ft-change in each dataset in red like in plot 6. You might not end up using this figure in your paper, but it could reveal some insightful patterns. 

### Powerpoint Slides and Paper
- [x] Remember that your powerpoint slides are due next Wednesday!
- [ ] After you get your abstract in, spend at least an hour each day outlining/researching/formatting these to get ready for the deadline next week. 
           - [ ] Tuesday
           - [ ]  Wednesday
           - [ ]  Thursday
           - [ ]  Friday

Misc.
- [x] Put code up on Github for steph and ben to see before end of day on Friday
- [x] Timesheet
- [x] Email about next weeks dr's appts
- [x] Email algae guy back
- [x] Start issue for next week

**Comment by mlkhorn** [comment]
@stephpenn1 I set this up as a tentative framework for this week. I'd love your input, especially about my plan for getting my abstract in. On Friday I took a look at another intern's "aspirational abstract" (she didn't have all her results yet either) and did some brainstorming on paper. In the next couple hours, I am looking to make some more progress on my abstract and talk with Ben about what I have so far when we meet at 12:30. Then if you and Skylar could give me some feedback on a completed draft I have done by our check in meeting at 4 AND/OR you would be willing to review what I have in google docs by late this afternoon, that could be be great. I would look to have a final draft complete by mid morning tomorrow.
